src/components/Breadcrumbs.js

```diff
@@ -22,10 +22,17 @@ export default class Breadcrumbs extends Component {
       const paths = splat.split('/');
       links = _.map(paths, (path, i) => {
         const before = (i == 0) ? '' : paths.slice(0, i).join('/') + '/';
-        return {
-          href: `${base}/${before}${path}`,
-          label: path
-        };
+        if (type == "data files") {
```

Comment:
> But we defined our routes so that individuals are called with their extensions.

Its not so according to [`master`](https://github.com/jekyll/jekyll-admin/blob/master/lib/jekyll-admin/server/data.rb#L11)
```ruby
get "/:path.?:ext?" do
```
(I vote for the more sensible resolution too)